ocaml/xapi/dbsync.ml

```diff
@@ -48,6 +48,14 @@ let create_host_metrics ~__context =
     )
     (Db.Host.get_all ~__context)
 
+let update_pool_recommendations ~__context =
```

Comment:
This will apply to the coordinator host only. So it can be moved into `dbsync_master.ml`.